supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -269,7 +269,9 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
       executionDir.toString.toFile.createIfNotExists(asDirectory = true, createParents = true)
 
       log.debug("{} Resolving job command", tag)
-      val command = localizeInputs(jobPaths.callRoot, runtimeAttributes.dockerImage.isDefined, fileSystems, jobDescriptor.inputs) flatMap {
+      val allowSoftLinks = configurationDescriptor.backendConfig.getBoolean("docker.allowSoftLinks")
+      val dockerLocEnabled = runtimeAttributes.dockerImage.isDefined && !allowSoftLinks
+      val command = localizeInputs(jobPaths.callRoot, dockerLocEnabled, fileSystems, jobDescriptor.inputs) flatMap {
```

Comment:
It would be great if you changed the `localizeInputs` method's `docker` parameter to be called just `allowSoftLinks` since that's what it really does.

If I'm reading this right, you're adding a short cut so that you can force it to allow soft-links even in docker mode. Is that what you wanted? A quick google brings up all sorts of "here be dragons" style warnings for doing that